packages/ihe-gateway-sdk/src/models/patient-discovery/patient-discovery-responses.ts

```diff
@@ -67,18 +67,21 @@ const outboundPatientDiscoveryRespDefaultSchema = baseResponseSchema.extend({
   patientId: z.string(),
 });
 
-const outboundPatientDiscoveryRespSuccessfulSchema = outboundPatientDiscoveryRespDefaultSchema
-  .merge(patientDiscoveryRespSuccessfulDefaultSchema)
-  .extend({
-    patientResource: inboundPatientResourceSchema.optional(),
+export const outboundPatientDiscoveryRespSuccessfulSchema =
+  outboundPatientDiscoveryRespDefaultSchema
+    .merge(patientDiscoveryRespSuccessfulDefaultSchema)
+    .extend({
+      patientResource: inboundPatientResourceSchema.optional(),
+    });
+
+export const outboundPatientDiscoveryRespFaultSchema =
+  outboundPatientDiscoveryRespDefaultSchema.extend({
+    patientMatch: z.literal(false).or(z.literal(null)),
   });
 export type OutboundPatientDiscoveryRespSuccessfulSchema = z.infer<
   typeof outboundPatientDiscoveryRespSuccessfulSchema
 >;
 
```

Comment:
Why moving these around? We've been keeping the type right after the schema to keep related stuff together. It would be good to have a good reason to change the pattern and update the code.

Reply:
no good reason. reverting it back